command_config.json

```diff
@@ -0,0 +1,52 @@
+{
+	"type":{		
```

Comment:
Indent properly please. Use 2 spaces if indentation gets too deep.

Reply:
Just format everything through black (python package).

Reply:
Sure, that's the plan #7

Reply:
Done.

Reply:
Note: black adds extra trailing comma while formatting JSON, which leads to errors while parsing using json in python.